src/workerd/api/node/zlib-util.h

```diff
@@ -86,7 +97,7 @@ class ZlibContext final {
 public:
   ZlibContext() = default;
 
-  KJ_DISALLOW_COPY(ZlibContext);
+  //KJ_DISALLOW_COPY(ZlibContext);
```

Comment:
nit... why comment this out? The `ZlibContext` likely shouldn't be copyable.

Reply:
I'm not finished yet. This was some crap for testing.